articles/operator-nexus/howto-configure-cluster.md

```diff
@@ -155,6 +155,70 @@ az networkcloud cluster deploy \
   --subscription "$SUBSCRIPTION_ID"
 ```
 
+> [!TIP]
+> To check the status of the `az networkcloud cluster deploy` command, it can be executed using the `--debug` flag.
+> This will allow you to obtain the `Azure-AsyncOperation` or `Location` header used to query the `operationStatuses` resource.
+> See the section [Cluster Deploy Failed](#cluster-deploy-failed) for more detailed steps.
+> Optionally, the command can run asynchronously using the `--no-wait` flag.
+
+### Cluster Deploy and Hardware Validation
+
+During a Cluster deploy process, one of the steps executed is hardware validation.
+The hardware validation procedure runs various test and checks against the machines
+provided through the Cluster's rack definition. Based on the results of these checks
+and any user skipped machines, a determination is done on whether sufficient nodes
+passed and/or are available to meet the thresholds necessary for deployment to continue.
+
+#### Cluster Deploy Action `--skip-validations-for-machines` parameter
```

Comment:
Suggest rephrasing to say "Deploy Cluster action with skipping specific bare-metal-machine"